tally_ho/apps/tally/static/js/barcode_verify.js

```diff
@@ -1,27 +1,60 @@
 function barcodes_match(empty_message, length_message, mismatch_message) {
     var barcode = document.getElementById('id_barcode');
```

Comment:
guess we should change these, https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
ooh i missed these, will make the changes, i should have used `const` for these.